forge-gui/res/adventure/common/world/enemies.json

```diff
@@ -6374,6 +6375,95 @@
 		"BiomeBlack"
 	]
 },
+	{
+		"name": "Drake",
+		"sprite": "sprites/enemy/dragon/drake.atlas",
+		"deck": [
+			"decks/standard/drake_ur_beginner.dck",
+			"decks/standard/drake_ur_mid.dck",
+			"decks/standard/drake_ur_unfair.dck"
+		],
+		"ai": "",
+		"flying": true,
+		"spawnRate": 0.95,
```

Comment:
Spawn rate calculations have been tweaked recently, so this won't have as much impact as it did previously. As of a few weeks ago, 0.95 would have resulted in this enemy being extremely rare to see in the wild if it ever spawned at all, on the order of 1/1000. Now this means that this enemy is a 5% lower chance to be chosen than an enemy with a 1.0 spawnRate.

So if your intent is for these to be super rare, this needs to be reduced significantly. (That's not a request, just making sure the new result is matching your intent).

Reply:
There was no intent here; I think I copy-pasted from the dragon or something and just switched some lines out as needed. Uncommon is probably appropriate, whatever number that would be.